SU-SWS/stanford_profile
# D8CORE-2201: adding extlink dependency

# Ready for review
Ready to merge, pending
https://github.com/SU-SWS/stanford_profile_helper/pull/11

# Summary
This installs the External Link module. It's needed to let AT know when you're pointing to an external link.

# Needed By (Date)
When possible

# Urgency
Addresses accessibility issues

# Steps to Test

1. re-install your site with this branch
2. Verify that the external link <svg> appears for all external links.

# Affected Projects or Products
D8Core

# Associated Issues and/or People
- JIRA ticket: https://stanfordits.atlassian.net/browse/D8CORE-2201
- Other PRs: https://github.com/SU-SWS/stanford_profile_helper/pull/11
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
